lib/translators/npm.js

```diff
@@ -82,7 +82,8 @@ class NpmTranslator {
 			on projects.
 			Although we could add the collections dependencies as project dependencies to the related modules
 		*/
-		if (!pkg.collection) {
+		const isCollection = typeof pkg.collection === "object";
```

Comment:
As discussed, I think we can even check for the `pkg.collection.modules` object being present

Reply:
Done